static/hub.js

```diff
@@ -99,6 +100,10 @@ function Hub(layout, subLangId, defaultLangId) {
         function (container, state) {
             return self.toolFactory(container, state);
         });
+    layout.registerComponent(Components.getToolInputViewWith().componentName,
```

Comment:
Maybe call `Components.getToolInputView()` instead so you're not missing the params?

Reply:
That makes sense. I had copy/pasted from above, but it begs the question: why is it calling `getToolViewWith()` instead of `getToolView()` (which actually doesn't exist)?  Should I change that too?

Reply:
Oh I see! Don't worry about the tool view (I didn't reaslise it's copy/paste). It's "ok" to call with no params, but in your case you _do_ have a non-param-taking version, so I guess you're OK :)